src/pages/s/index.tsx

```diff
@@ -70,6 +98,33 @@ function ArticlesSection() {
   return <></>;
 }
 
+interface Partner {
```

Comment:
no need for defining this type if `partnersData` is within the PartnersSection function.